linux/debug_pty.go

```
@@ -0,0 +1,154 @@
+	log.Printf("Command setup: %s, Args: %v, Dir: %s", cmd.Path, cmd.Args, cmd.Dir)
+
+	// Start PTY
+	ptmx, err := pty.Start(cmd)
+	if err != nil {
+		log.Printf("Failed to start PTY: %v", err)
+		return
+	}
+	defer func() {
+		ptmx.Close()
+		if cmd.Process != nil {
+			cmd.Process.Kill()
```

golangci-lint (1.64.8)

Error return value of `cmd.Process.Kill` is not checked

(errcheck)